src/command/use.rs

```diff
@@ -1,29 +1,39 @@
 use structopt::StructOpt;
 
 use crate::command::Command;
+use notion_core::error::ErrorDetails;
 use notion_core::session::{ActivityKind, Session};
 use notion_fail::{ExitCode, Fallible};
 
-#[macro_export]
-macro_rules! usage {
-    () => {
-        "notion-use
+// NOTE: These use the same text as the `long_about` in crate::cli.
+//       It's hard to abstract since it's in an attribute string.
+
+pub(crate) const USAGE: &'static str = "The subcommand `use` is deprecated.
 
-DEPRECATED:
     To install a tool in your toolchain, use `notion install`.
     To pin your project's runtime or package manager, use `notion pin`.
-"
-    };
-}
+";
+
+const ADVICE: &'static str = "
+    To install a tool in your toolchain, use `notion install`.
+    To pin your project's runtime or package manager, use `notion pin`.
+";
 
 #[derive(StructOpt)]
-pub(crate) struct Use {}
+pub(crate) struct Use {
+    #[allow(dead_code)]
+    anything: Vec<String>,
```

Comment:
Might be worth a comment that this prevents StructOpt from showing an argument error when you do `notion use node 10`, instead just showing the deprecation error.
```suggestion
    anything: Vec<String>, // Prevent StructOpt argument errors when invoking e.g. `notion use node`
```